cmd/apps.go

```diff
@@ -85,8 +86,14 @@ func AppsList(results int) error {
 
 	fmt.Printf("=== Apps%s", limitCount(len(apps), count))
 
+	var appNames []string
```

Comment:
Aha did not know you could do that in Go.

I have to say I found my method a bit cumbersome but I did not know how to write a new Sort function. I did not realise it was as simple as implementing a simple (implicit) interface.

Its on my todo list to learn Go properly btw, just did not have the time or use case yet :)